src/src/pages/Dashboard/components/QuickStart/QuickStart.tsx

```diff
@@ -28,24 +31,24 @@ function QuickStart() {
           tint={100}
           className='QuickStart__section__title'
         >
-          Integrate Aim with your code
+          Integrate FastTrackML with your code
         </Text>
         <CodeBlock
-          code={`from aim import Run
+          code={`from mlflow import log_metric, log_params, set_tracking_uri
```

Comment:
it would be more idiomatic to change the line to `import mlflow` and use the fully-qualified names below